presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PruneUnreferencedOutputs.java

```diff
@@ -318,7 +318,7 @@ public PlanNode visitAggregation(AggregationNode node, RewriteContext<Set<Symbol
                     if (aggregation.getMask().isPresent()) {
                         expectedInputs.add(aggregation.getMask().get());
                     }
-                    aggregations.put(symbol, new Aggregation(call, aggregation.getSignature(), aggregation.getMask()));
+                    aggregations.put(symbol, new Aggregation(call, aggregation.getSignature(), aggregation.getMask(), aggregation.getOrderBy(), aggregation.getOrdering()));
```

Comment:
Consider adding check state here that order by symbols are are part of `expectedInputs`